src/options/SchemaBuilderOptions.ts

```diff
@@ -44,6 +44,22 @@ export interface SchemaBuilderOptions extends GraphModule {
      */
     customTypeDefs?: any;
 
+    /**
+     * Custom TypeORM ConnectionOptionsReader options.
+     */
+    customTypeORMReaderOptions?: {
```

Comment:
We'll need a better naming here. Not sure how to organize it yet, for now we can do:

```
{
   ...
   typeorm: {
           connectionOptionsReaderOptions?: { ... },
           connectionName?: string // "default" by default
    }
}
```